### ISSUE TITLE
Sometimes <Link>'s don't work in dev

### ISSUE THREAD

**Issue by bgold0** [zeit/next.js#5772]
# Bug report

## Describe the bug
Sometimes when I have links they don't want to load in my dev environment. However when it's on the cloud it works fine. I'm not sure why.

## To Reproduce
Don't really know what steps I can do. Sometime's it works, sometime's it doesn't. If I refresh the page in a new tab, it will work as expected.

`<Link href="/config/link"><a>Text Here</a></Link>`

## Expected behavior
For it to go to it's respected link.

## System information
 - OS: macOS
 - Browser: safari
 - Version of Next.js: `"next": "^7.0.2",`

**Comment by iamandyk** [comment]
I think this might be a duplicate of https://github.com/zeit/next.js/issues/5598

**Comment by malimccalla** [comment]
If this is the same issue as #5590 see this workaround https://github.com/zeit/next.js/issues/5598#issuecomment-439678578

**Comment by rodrigooler** [comment]
I was able to solve my problem, it was basically a Provider that was giving problem to render the page.

**Comment by rodrigooler** [comment]
@iamandyk yes, in my case I am using a redux alternative that is the reworm and it has a provider that I put in the _app ... In this both the <Link href = href /> and the Router.push have stopped working. When removing returned to work normally, hence my solution was to instantiate the provider inside a HOC and only call when it will use. The route system has returned to normal.
